test-e2e/members.js

```diff
@@ -25,8 +25,7 @@ test('getting yourself after creating project', async (t) => {
   const deviceInfo = manager.getDeviceInfo()
   const project = await manager.getProject(await manager.createProject())
 
-  const me = await project.$member.getById(project.deviceId)
-
+  const { joinedAt: _, ...me } = await project.$member.getById(project.deviceId)
```

Comment:
all tests fail 'cause they don't expect a `joinedAt` wondering options:
1. remove `joinedAt` from the comparison and leave it at that
2. remove `joinedAt` from the comparison but assert that the field should be there (despite not caring about matching values)
3. another way of getting the expected value for `joinedAt` to add to the expected object

Reply:
I like the 2nd option. Maybe we could also check that the timestamp is near the current timestamp?